### PR TITLE
Feature/drupal 10.1.8

### PR DESCRIPTION
**Pull request by Mr-Martin-Kristiansen**
Updated ReadMe.
Updated packages.
Updated drupal to 10.1.8.

### CODE REVIEW

**Review by martinyde — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -183,9 +183,6 @@
 
 # ------------------------------------------------------------------------------
 
-# We want all custom site settings in settings.local.php
-!/web/sites/*/settings*.php
-
```
Comment (rimi-itk) [inline .gitignore:188]: I don't understand why this change is needed in relation to the other changes in this pull request. Can you elaborate?
